libraries/AP_HAL_ChibiOS/hwdef/F4BY_F427/hwdef-bl.dat

```diff
@@ -0,0 +1,42 @@
+# hw definition file for processing by chibios_hwdef.py
+# for f4by bootloader
+
+MCU STM32F4xx STM32F427xx
+
+APJ_BOARD_ID 21
```

Comment:
You'll need to allocate another board ID in `Tools/AP_Bootloader/board_list.txt`